workers/tasks/export/SimpleNotesList.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Moved from `components` — it's too tightly coupled to the export code to be work shooting for reusability, IMO.